packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImageResizer.java

```diff
@@ -12,7 +12,30 @@
 import java.io.IOException;
 
 class ImageResizer {
-  File resizedImage(String path, Double maxWidth, Double maxHeight) throws IOException {
+  private final ExifDataCopier exifDataCopier;
+
+  ImageResizer(ExifDataCopier exifDataCopier) {
+    this.exifDataCopier = exifDataCopier;
+  }
+
+  String resizeImageIfNeeded(String imagePath, Double maxWidth, Double maxHeight) {
```

Comment:
Can you document what this method returns (e.g. the path of the rescaled image or the original path if no scalling was necessary)?